Creating Tests for Task 20

vitaliesirbu opened the pull request.

vitaliesirbu committed:
Creating Tests for Task 20

shukal94 reviewed (changes requested):
1) exclude test-output dir from PR
2) move testng suites into src/test/resources dir
3) package name should be named in lower-case format (Unit -> junit)

vitaliesirbu committed:
Add excluding rule for "test-output" folder;
Moving testng suit to resources directory;
Renaming package names to lowercase.

vitaliesirbu committed:
Review changes.

shukal94 commented on src/test/java/testng/CartTest.java:
```diff
@@ -0,0 +1,47 @@
+    public void tearDown() {
+        // Optionally, you can assert all in tearDown to ensure assertions are checked
+        // even if there's an exception in the test method before reaching softAssert.assertAll()
+        softAssert.assertAll();
```
with the increasing number of the tests (in average test project could contain 500+ test classes) it's easy to lose this soft assert which could cause false-positive results.
I recommend to define soft assertion and soft assert all() call for each test method

vitaliesirbu committed:
Adapting softAssert pattern.

vitaliesirbu committed:
Deleting comments
